terraform/bangladesh/main.tf

```diff
@@ -27,110 +27,6 @@ provider "aws" {
   profile = "bangladesh"
 }
 
-#
```

Comment:
2 Ansible jobs seem to be outdated: update_bangladesh_demo_config.yml & update_bangladesh_production_config.yml

Reply:
Can remove bangladesh from hosts in ansible/README